src/pages/Validators/StakingDrawer.tsx

```diff
@@ -0,0 +1,127 @@
+import {
+  Button,
+  Divider,
+  Link,
+  List,
+  ListItem,
+  Typography,
+  useTheme,
+} from "@mui/material";
+import * as React from "react";
+import SideDrawer from "../../components/SideDrawer";
+import {REWARDS_LEARN_MORE_LINK} from "./Components/Staking";
+
+type StakingDrawerProps = {
+  open: boolean;
+  handleClick: () => void;
+};
+
+export function StakingDrawer({open, handleClick}: StakingDrawerProps) {
+  const theme = useTheme();
+
+  return (
+    <SideDrawer open={open} toggleSideDrawer={handleClick}>
+      <List sx={{alignItems: "flex-start", margin: 3}}>
+        <Typography variant="h4" sx={{margin: 1, marginBottom: 3}}>
+          Delegated Staking FAQ
+        </Typography>
+        <Divider flexItem sx={{marginY: 1}} />
+        <ListItem
+          sx={{
+            flexDirection: "column",
+            alignItems: "flex-start",
+            padding: 1,
+            color: theme.palette.mode === "dark" ? "#D4D4D4" : null,
```

Comment:
In ANS I usually see we pull color from mui material like `colors.grey[500]` and the color is set in the color scheme as a central place? do you think we should do that here too?